README.md

```diff
@@ -1,84 +1,184 @@
+
+Run the following command:
 
--   **Fast & flexible compilation pipeline**
-    -   Automatic Solidity compiler version detection & installation (under `~/.svm`)
-    -   **Incremental compilation & caching**: Only changed files are re-compiled
-    -   Parallel compilation
-    -   Non-standard directory structures support (e.g. [Hardhat repos](https://twitter.com/gakonst/status/1461289225337421829))
--   **Tests are written in Solidity** (like in DappTools)
--   **Fast fuzz testing** with shrinking of inputs & printing of counter-examples
--   **Fast remote RPC forking mode**, leveraging Rust's async infrastructure like tokio
--   **Flexible debug logging**
-    -   DappTools-style, using `DsTest`'s emitted logs
-    -   Hardhat-style, using the popular `console.sol` contract
--   **Portable (5-10MB) & easy to install** without requiring Nix or any other package manager
--   **Fast CI** with the [Foundry GitHub action][foundry-gha].
+```bash
+cargo install --path ./crates/forge --profile local --force --locked
+```
 
-### How Fast?
+### Installing `cast` 📡
 
-Forge is quite fast at both compiling (leveraging [ethers-solc][ethers-solc]) and testing.
+Similarly, to install `cast`, use the following command. Like `forge`, this will replace any existing `cast` installations without altering functionality. Once installed, `cast` becomes available as an executable in `~/.cargo/bin`.
 
-See the benchmarks below. More benchmarks can be found in the [v0.2.0 announcement post][benchmark-post] and in the [Convex Shutdown Simulation][convex] repository.
+Run the following command:
 
-**Testing Benchmarks**
+```bash
+cargo install --path ./crates/cast --profile local --force --locked
+```
 
-| Project                            | Forge | DappTools | Speedup |
-| ---------------------------------- | ----- | --------- | ------- |
-| [transmissions11/solmate][solmate] | 2.8s  | 6m34s     | 140x    |
-| [reflexer-labs/geb][geb]           | 0.4s  | 23s       | 57.5x   |
-| [Rari-Capital/vaults][vaults]      | 0.28s | 6.5s      | 23x     |
+### Installing the Entire Suite 📦
 
-_Note: In the above benchmarks, compilation was always skipped_
+To install all the tools in the suite:
 
-**Compilation Benchmarks**
+```bash
+cargo build --release
+```
 
-<img alt="Compilation benchmarks" src=".github/compilation-benchmark.png" width="693px" />
+## Quickstart 
 
-**Takeaway: Forge compilation is consistently faster by a factor of 1.7-11.3x, depending on the amount of caching involved.**
+In an empty directory, run the following command:
+``` 
+forge init
+```
 
-## Cast
+Let's check out what forge generated for us:
 
-Cast is a swiss army knife for interacting with Ethereum applications from the command line.
+```
+$ cd hello_foundry
+$ tree . -d -L 1
+.
+├── cache
+├── lib
+├── out
+├── script
+├── src
+├── test
+└── zkout
```

Comment:
After `forge init` you will find README.md  foundry.toml  lib  script  src  test in the directory.

cache, out, zkout are generated after `forge build --zksync`